CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## UNRELEASED
+## v0.20.1 (July ??, 2019)
+
+BUG FIXES:
+
+* Fixed issue with templates not rendering with `-once` [GH-1227, GH-1196, GH-1207]
```

Comment:
Something we do in Consul is to link directly to the GH issues from the changelog. 

I'm seeing that it hasn't been done in consul-template, but it could be nice to add to future releases. Small speed-up for users who want to jump to the issue/PR.

eg:
```suggestion
* Fixed issue with templates not rendering with `-once` [[GH-1227] 	(https://github.com/hashicorp/consul-template/pull/1227)], [[GH-1196] 	(https://github.com/hashicorp/consul-template/issues/1196)], [[GH-1207](https://github.com/hashicorp/consul-template/issues/1207)]
```

Reply:
Thanks!